hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -573,11 +581,15 @@ https://hazelcast.org/documentation/.
         * <capacity>:
             Capacity of the scheduled executor. The default value is 100.
             This is the maximum number of tasks for a given scheduled executor on a particular partition.
+        * <quorum-ref>:
+            Adds the Split Brain Protection for this queue which you configure using the <quorum> element. You should set the <quorum-ref>'s value
```

Comment:
this seems a copy&paste error all over the comments: `Adds the Split Brain Protection for this queue`, indeed this not a  queue?

Reply:
true, will fix that, thx!

Reply:
Fixed.